lib/rucio/db/sqla/constants.py

```diff
@@ -199,3 +199,8 @@ class SubscriptionState(Enum):
 class TransferLimitDirection(Enum):
     SOURCE = 'S'
     DESTINATION = 'D'
+
+
+class DBSessionOperation(Enum):
```

Comment:
Wouldn't this rather be

`class TransactionType(Enum)`

?

Reply:
so resulting in something like

`with db_session(TransactionType.WRITE) as session:`
`with db_session(TransactionType.READ) as session:`

Reply:
I think `OperationType` is a bit too vague, as it's not clear that it's about database operations. I propose we meet in the middle and use `DatabaseOperationType`, which is also in line with the terminology we use here https://rucio.github.io/documentation/operator/database/ - what do you think?